cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -350,7 +355,7 @@ private void processCapacityEntries(CruiseControlSpec spec, Set<NodeRef> kafkaBr
         // We create a capacity for each broker node
         for (NodeRef node : kafkaBrokerNodes)   {
```

Comment:
Have you considered merging this loop and the if/loop below into one? I would assume you can get through each node in this loop and check the overrides directly in there instead of first doing X for every node and then possibly throwing it away in the next code block if the overrides are set. Or is there really an reason to have it separate?

Reply:
They are now merged into one.